app/_src/gateway-operator/guides/konnect-plugin-binding.md

```diff
@@ -0,0 +1,293 @@
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: cp
+' | kubectl apply -f - 
+```
+
+Create a consumer:
+
+```bash
+echo '
+kind: KongConsumer
+apiVersion: configuration.konghq.com/v1
+metadata:
+  namespace: default
+  name: consumer-plugin-binding-combination
+username: consumer-test
+spec:
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: cp
+' | kubectl apply -f -
+```
+
+Create a plugin:
+
+```bash
+echo '
+kind: KongPlugin
+apiVersion: configuration.konghq.com/v1
+metadata:
+  namespace: default
+  name: rate-limiting-minute-10
+plugin: rate-limiting
+config:
+  policy: local
+  minute: 10
+' | kubectl apply -f - 
+```
+
+Then, you can create a `KongPluginBinding` including both references to the `KongService` and the `KongCosumer` to attach the plugin to the service and the consumer:
+
+```bash
+echo '
+kind: KongPluginBinding
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  namespace: default
+  name: binding-combination-service-consumer
+spec:
+  pluginRef:
+    kind: KongPlugin
+    name: rate-limiting-minute-10
+  targets:
+    serviceRef:
+      group: configuration.konghq.com
+      kind: KongService
+      name: service-plugin-binding-combination
+    consumerRef:
+      name: consumer-plugin-binding-combination
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: cp
+' | kubectl apply -f - 
+```
+
+### Using Annotations to Bind Plugins to Other Entities
+
+You can also use the `konghq.com/plugins` annotation to attach plugins to other entities like what we used in {{ site.kic_product_name }}. The {{ site.kgo_product_name }} will create `KongPluginBinding` resources for the annotations and configure them in {{ site.konnect_short_name }}.
+
+For the example above, you can create a `KongPlugin` and a `KongService` like this:
+
+```bash
+echo '
+kind: KongPlugin
+apiVersion: configuration.konghq.com/v1
+metadata:
+  namespace: default
+  name: rate-limiting-minute-10
+plugin: rate-limiting
+config:
+  policy: local
+  minute: 10
+' | kubectl apply -f - 
+```
+
+```bash
+echo '
+kind: KongService
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  namespace: default
+  name: service-example
+  annotations:
+    konghq.com/plugins: rate-limiting-minute-10
+spec:
+  host: example.com
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: cp
+' | kubectl apply -f -
+```
+
+Then you can also see the plugin is attached to the service in {{ site.konnect_short_name }}. You can also check the `KongPluginBinding` resource by `kubectl get kongpluginbindings`. You can see the created `KongPluginBinding` like:
+
+```
+kubectl get kongpluginbinding
+NAME                            PLUGIN-KIND   PLUGIN-NAME                  PROGRAMMED
+rate-limiting-minute-10-a0z1x   KongPlugin    rate-limiting-minute-10      True
+```
+
+#### Attaching Plugins to Multiple Entities
```

Comment:
I'd add a table to this section showing what combinations of entities will be produced when using `konghq.com/plugins` annotation.

This can be based on https://github.com/Kong/gateway-operator/blob/f189f2d3972fe08122bb27ead06152132ee791a0/controller/konnect/reconciler_kongplugin_combinations_test.go.

So for instance if service and route have the annotation we produce 2 plugins: 1 for service and 1 for route.

When service and consumer have the annotation: we produce 1 plugin binding both to the plugin.